app/Migrations/Version20250212071939.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20250212071939 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->createTable('mails_sent');
+
+        $table->addColumn('identifier', Types::GUID)->setLength(36)->setNotnull(true);
+        $table->addColumn('email', Types::STRING)->setLength(100)->setNotnull(true);
+        $table->addColumn('type', Types::STRING)->setLength(100)->setNotnull(true);
+        $table->addColumn('dateTime', Types::DATETIME_IMMUTABLE)->setNotnull(true);
+
+        $table->setPrimaryKey(['identifier', 'type']);
```

Comment:
I think this can result in issues because for one ownership there can be 2 emails, one for the created and one for the approved.

This also makes me wonder if we need to be more explicit about the column names instead of having these generic column names.

Reply:
1) This is why Type is included in the primary key, so a record will read something like
id-of-the-ownership-request - request_approved - koen@publiq.be - some date

2) I can see that, I would really like to try if we can make do with generic names because I don't want to end up in a situation where we start sending more mails from UDB and we end up with multiple tables all tracking sent emails.

Reply:
Ah, sorry, I missed the type part.